app/templates/_brief_q_and_a.html

```diff
@@ -11,25 +11,11 @@
   </a>
 {% endif %}
 
-{{ summary.heading("Questions asked by suppliers", id="clarification-questions") }}
-{% call(question) summary.list_table(
-  brief.clarificationQuestions,
-  caption="Questions asked by suppliers",
-  field_headings=[
-    "Supplier question",
-    "Buyer answer"
-    ],
-    field_headings_visible=False,
-    empty_message="No questions have been answered yet"
-) %}
-  {% call summary.row() %}
-    {% call summary.field(first=True, wide=False) -%}
-      <span aria-label="question">{{ question.number }}.</span>
-      {{ question.question | format_links | preserve_line_breaks }}
-    {%- endcall %}
-    {{ summary.text(question.answer | format_links | preserve_line_breaks) }}
-  {% endcall %}
-{% endcall %}
+<h2 id="clarification-questions" class="govuk-!-font-size-27 govuk-!-padding-top-6">Questions asked by suppliers</h2>
+{% if brief.clarificationQuestions is undefined or brief.clarificationQuestions|length == 0 %} 
+<p class="summary-item-no-content">No questions have been answered yet</p>
+{% endif %}
+{{ govukSummaryList({"rows": brief.clarificationQuestions}) }}
```

Comment:
Is there something we should be doing to make sure that the summary list and the heading are connected in HTML? Like a legend?

Reply:
I don't think so. There's nothing like that in [this example](https://design-system.service.gov.uk/patterns/check-answers/), for instance. 

We could potentially slap an `aria-labelledby` or similar onto the summary list, but I think that would just be repeating the `<h2>` info.

Reply:
Okay no worries then